home/api.py

```diff
@@ -42,7 +43,7 @@ def list(self, request, *args, **kwargs):
         super(ContentPagesViewSet, self).list(self, request, *args, **kwargs)
 
     def get_queryset(self):
-        queryset = super(ContentPagesViewSet, self).get_queryset()
+        queryset = ContentPage.objects.live()
```

Comment:
@codiebeulaine if we're not using `get_queryset` then will we still be able to use the other built-in filters? I see that `get_queryset` calls [`get_base_queryset`](https://github.com/wagtail/wagtail/blob/main/wagtail/api/v2/views.py#L491), so if we're skipping `get_queryset` what are  losing out on?